### ISSUE TITLE
AccessDeniedListener tag does not get cleared (wrong tag name)

### ISSUE THREAD

**Issue by NinoFloris** [FriendsOfSymfony/FOSRestBundle#1345]
Tried to use my own service for the acces_denied_listener config, the FOSRest listener is still registered.

Fix is to change

```php
if (!empty($config['access_denied_listener']['service'])) {
   $service = $container->getDefinition('fos_rest.access_denied_listener');
   $service->clearTag('kernel.event_listener');
}
```

into: 

```php
if (!empty($config['access_denied_listener']['service'])) {
   $service = $container->getDefinition('fos_rest.access_denied_listener');
   $service->clearTag('kernel.event_subscriber');
}
```

As the AccessDeniedListener service is really a kernel subscriber, not a listener.

Should I submit a pull request?

**Comment by Ener-Getick** [comment]
IMO we should update all our listeners to kernel subscribers. 
And if you can please submit a pull request ☺

**Comment by xabbuh** [comment]
see #1348

**Comment by Ener-Getick** [comment]
Fixed in #1348
